src/OrleansRuntime/Catalog/Catalog.cs

```diff
@@ -19,6 +19,8 @@
 
 namespace Orleans.Runtime
 {
+    using Orleans.Runtime.Messaging;
```

Comment:
Should this be before or after `namespace Orleans.Runtime`? I'm note sure if we should maintain consitency of keeping namespace directives on the other scope.